.github/workflows/python-package.yml

```diff
@@ -26,16 +26,16 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install flake8 pytest
-        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        pip install -r requirements-dev.txt
     - name: Lint with flake8
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    - name: Test with pytest
+    - name: Test with pytest and report coverage
       run: |
         cd tests
-        pytest
+        coverage run -m pytest
+        coverage report -m
```

Comment:
How is the coverage report accessed by the users?

Reply:
As configured you have to open up the Actions workflow log (e.g., [this one from my branch](https://github.com/elbeejay/dataretrieval-1/actions/runs/3252613229/jobs/5339010780)). Then you can look by expanding the test/coverage section as in the screenshot below. Integration with codacy or codecoverage would produce a "richer" output, but I wasn't sure if either of those third-party connections were going to be appropriate for this project, and in either case they require some configuration by a maintainer. 


<img width="1906" alt="Screen Shot 2022-10-17 at 10 20 07 AM" src="https://user-images.githubusercontent.com/1770513/196202039-edc7908f-f45d-49e9-8728-4363b6f60f85.png">